userdocs/src/usage/addons.md

```diff
@@ -118,6 +118,31 @@ eksctl utils describe-addon-versions --kubernetes-version 1.22 --types "infra-ma
 ```
 The `types`, `owners` and `publishers` flags are optional and can be specified together or individually to filter the results.
 
+# TODO: update documentation here after merging DescribeAddonConfiguration PR
```

Comment:
Reminder to update this after merging https://github.com/weaveworks/eksctl/pull/6056.